staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/controller.go

```diff
@@ -286,6 +287,10 @@ func (e *quotaEvaluator) checkQuotas(quotas []corev1.ResourceQuota, admissionAtt
 		}
 
 		if err := e.quotaAccessor.UpdateQuotaStatus(&newQuota); err != nil {
+			// If there is a transient update error, it doesn't count against the retry quota
+			if isRetryableError(err) {
+				remainingRetries = retries
+			}
```

Comment:
No need to introduce an extra function, from the examples I found in the code base `apierrors.IsConflict` is used  inline

https://github.com/search?q=repo%3Akubernetes%2Fkubernetes+apierrors.IsConflict&type=code

```suggestion
			if apierrors.IsConflict(err) {
				remainingRetries = retries
			}
```

Reply:
Yeah the idea was to have additional retryable errors but maybe doesn't matter

Reply:
From the examples I found in the code base, the usual pattern with additional retryable errors is something like `if (first type of errors) || (second type of errors) || ...`